docs/en/reference/sql/dql/WINDOW_CLAUSE.md

```diff
@@ -243,13 +258,24 @@ WINDOW w1 AS (PARTITION BY col1 ORDER BY col5 ROWS_RANGE BETWEEN 10s PRECEDING A
 
 ![Figure 5: window exclude current time](../dql/images/window_exclude_current_time.png)
 
-### Window Frame Max Size
+### 3. Window Exclude Current Row
```

Comment:
give an example of exculde current row?

Reply:
yes, I'm asking @michelle-qinqin for the favor 🌝

Reply:
the images have been added in this PR: 
https://github.com/4paradigm/OpenMLDB/pull/2297